<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Disass</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/Artwork/Icons/Logo.png</normaloff>:/Artwork/Icons/Logo.png</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="sendButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="pidBox">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>PID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>540</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="commandBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Command</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="atNtButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string notr="true">ATnT Syntax</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="intelButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Intel Syntax</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="codeOutputTabs">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>591</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="assemblyOutput">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>Disassembly</string>
     </attribute>
     <widget class="QTextBrowser" name="gdbAsmOutputBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="codeOutput">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>Code</string>
     </attribute>
     <widget class="QTextBrowser" name="gdbCodeOutputBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gdbOutput">
     <attribute name="title">
      <string>GDB Output</string>
     </attribute>
     <widget class="QTextBrowser" name="gdbOutputBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="breakButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Break</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="breakBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Address/Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stepIntoButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>30</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Step Into</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stepOverButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>0</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Step Over</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continueButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>90</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextCodeLineButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>60</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Next Code Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Quit Debugging</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="commandLineArgumentsBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>540</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Program Arguments</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="registerBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>341</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextBrowser" name="stackBox">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>0</y>
      <width>141</width>
      <height>581</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="fileArchBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>60</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>File Architecture</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1115</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionLicense"/>
    <addaction name="actionContribute"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Select Executable</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License and Credit</string>
   </property>
  </action>
  <action name="actionContribute">
   <property name="text">
    <string>Contribute</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
